concatenate_ice/concat_ice_daily.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+#concatenate sea-ice daily output
+#script inspired from https://github.com/COSIMA/1deg_jra55_ryf/blob/master/sync_data.sh#L87-L108
+
+out_dir=$(ls -td archive/output??? | head -1)/ice/OUTPUT #latest output dir only
```

Comment:
```suggestion
out_dir=$(ls -dr archive/output??? | head -1)/ice/OUTPUT #latest output dir only
```

Reply:
We need to support 4 digit output directory numbers, and sorting must work even though they are only padded to 3 digits, which is possible with `-v` e.g.

```
$ ls -drv /g/data/ik11/outputs/access-om2-01/01deg_jra55v13_ryf9091/output* | head -1
/g/data/ik11/outputs/access-om2-01/01deg_jra55v13_ryf9091/output1115
```

Reply:
Thanks done